org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/text/JavaSourceViewerConfiguration.java

```diff
@@ -457,6 +461,9 @@ public IContentAssistant getContentAssistant(ISourceViewer sourceViewer) {
 			ContentAssistProcessor javadocProcessor= new JavadocCompletionProcessor(getEditor(), assistant);
 			assistant.setContentAssistProcessor(javadocProcessor, IJavaPartitions.JAVA_DOC);
 
+			ContentAssistProcessor markdownProcessor= new JavadocCompletionProcessor(getEditor(), assistant);
+			assistant.setContentAssistProcessor(javadocProcessor, IJavaPartitions.JAVA_MARKDOWN_COMMENT);
```

Comment:
did you mean to pass markdownProcessor instead of javadocProcessor?

Reply:
Thanks, it's fixed now. Does it mean you have reviewed the patch?

Reply:
> Does it mean you have reviewed the patch?

I wouldn't call it a review. I was curious about what changes were needed, and stumbled upon this little glitch as a random finding :)